app/src/main/java/org/sagebionetworks/research/mpower/profile/ProfileFragment.java

```diff
@@ -48,4 +61,16 @@ public void onDestroyView() {
         super.onDestroyView();
         unbinder.unbind();
     }
+
+    @OnClick(R.id.licenses_button)
+    public void onLicensesClicked() {
+        LOGGER.debug("Clicked on the licenses button");
+        String path =  MpResourceManager.getInstance().getLicense().getAbsolutePath();
+        android.content.Intent intent = org.researchstack.backbone.ui.ViewWebDocumentActivity.newIntentForPath(this.getActivity(),

```

Comment:
It may have been a copy/paste issue, but no need for full package names for class types.  If you delete "android.content" then Android Studio will highlight "Intent" in red and underline it.  Then put the cursor on "Intent" and on your keyboard press "Option + Enter" and it will auto-import the class for you.